identity/webapp/src/utility/api-caller.ts

```diff
@@ -14,7 +14,10 @@ const identityInstance: AxiosInstance = axios.create({
   },
 });
 
-type ContextState = ApplicationState | { user: { accessToken: string } };
+type ContextState = ApplicationState | { user?: { accessToken?: string } };
```

Comment:
Would we ever get a user who doesn't have an access token?

Reply:
Looks like we do at [this level](https://github.com/wellcomecollection/wellcomecollection.org/blob/main/identity/webapp/src/utility/configure-auth0.ts#L29), which looks like the types need tightening up there.

Reply:
Just to clarify, I understand this to mean "would we ever have a `user` but not a `user.accessToken`?" - no, I'll see if that can be reflected in the types.